.circleci/config.yml

```diff
@@ -175,8 +175,12 @@ jobs:
           name: Update the images we're using
           command: |
             cd ./monitor-ci && make update && make build NODE=cypress && make build NODE=ingress && make build NODE=dex
+            docker tag local/cypress:latest quay.io/influxdb/ui-e2e:${CIRCLE_SHA1}
+            docker tag local/cypress:latest quay.io/influxdb/ui-e2e:latest
             docker load < ~/docker-cache/image.tar
             docker tag quay.io/influxdb/ui-acceptance:${CIRCLE_SHA1} quay.io/influxdb/ui-acceptance:latest
+            docker push quay.io/influxdb/ui-e2e:${CIRCLE_SHA1}
+            docker push quay.io/influxdb/ui-e2e:latest
```

Comment:
do we want to tag latest to every PR? wont that cause collisions?

Reply:
thanks! i had "only do this in master" in there at some point- must have removed?